### PR TITLE
Fix timestamp format

### PR DESCRIPTION
**Pull request by PuruVJ**
## Description
Time format has been fixed

## Attach Screenshot
image

### CODE REVIEW

File: twindle-cli/package.json [hunk 1]
```diff
@@ -32,7 +33,7 @@
     "puppeteer": "^5.4.1",
     "smtp-client": "^0.3.3",
     "string.prototype.matchall": "^4.0.2",
-    "yargs": "^16.1.0",
-    "twemoji": "13.0.1"
+    "twemoji": "13.0.1",
```
Comment (proful) [inline twindle-cli/package.json:36]: why? 😊

File: twindle-cli/twitter/utils/tweet-utils.js [hunk 2]
```diff
@@ -32,7 +33,7 @@ const getUserObject = (responseJSON) => responseJSON.includes.users[0];
 const createCustomTweet = (tweet_object, user_object) => {
   return {
     id: tweet_object.id,
-    createdAt: tweet_object.created_at,
+    createdAt: format(new Date(tweet_object.created_at), "MMM d, yyyy"),
```
Comment (proful) [inline twindle-cli/twitter/utils/tweet-utils.js:36]: Please add AM or PM
e.g., 5 PM

File: twindle-cli/package.json (hunk 1 above)
Comment (PuruVJ) [reply twindle-cli/package.json:36]: Twemoji?

File: twindle-cli/twitter/utils/tweet-utils.js (hunk 2 above)
Comment (PuruVJ) [reply twindle-cli/twitter/utils/tweet-utils.js:36]: Ok

File: twindle-cli/package.json (hunk 1 above)
Comment (proful) [reply twindle-cli/package.json:36]: You just changed the order?

There is no need for any package json changes

File: twindle-cli/twitter/utils/tweet-utils.js (hunk 2 above)
Comment (PuruVJ) [reply twindle-cli/twitter/utils/tweet-utils.js]: Done

File: twindle-cli/package.json (hunk 1 above)
Comment (PuruVJ) [reply twindle-cli/package.json:36]: I didn't change order. It just happened.

Maybe directly doing `npm i package` reorders as alphabetical.

Comment (proful) [reply twindle-cli/package.json:36]: Remove package json related changes

**Comment by PuruVJ** [comment]
Updated 
image

**Review by proful — CHANGES_REQUESTED** [review]
These files package json and lock files not needed to be part of PR

**Comment by PuruVJ** [comment]
Which ones?

**Comment by PuruVJ** [comment]
@proful The global package-lock.json is better deleted, cuz it was there regardless of absense of package.json. And all package-lock.json should be committed. I've read that this file is intended for that.

And I used a package to to format the date, so yes, we need all these changes

**Comment by proful** [comment]
@PuruVJ merged this time. Next time don't add all.

Add only files essential

**Comment by PuruVJ** [comment]
@proful Which ones do you think here are necessary files?

**Comment by proful** [comment]
Usually package json and lock files no need to add. If you not added any dependency.

**Comment by johnjacobkenny** [comment]
Hey @PuruVJ how did you fix the extra commits issue

**Pull request merged**